test/Microsoft.AspNet.Mvc.FunctionalTests/TestHelper.cs

```diff
@@ -114,7 +114,9 @@ public static TestServer CreateServer(Action<IApplicationBuilder> builder, strin
                 applicationBasePath,
                 applicationWebSiteName);
             services.AddInstance<IApplicationEnvironment>(environment);
```

Comment:
Perhaps replacing the application environment might no longer be needed now that hosting enviroment doesn't depend on it